tests/cancun/eip5656_mcopy/test_mcopy_contexts.py

```diff
@@ -189,7 +175,7 @@ def test_no_memory_corruption_on_upper_call_stack_levels(
 
 
 @pytest.mark.parametrize(
-    "opcode",
+    "call_opcode",
```

Comment:
this is clearly abusing the `call_opcode` parameter, is there pytest magic which could make it more elegant (combine `call_opcode` with `[CREATE, CREATE2]` to become `opcode`)?

Reply:
I think you need another marker for this (sorry about that), it's in an open PR (#739) and it's called `with_all_create_opcodes` so it parametrizes `create_opcode` with `[Op.CREATE, Op.CREATE2, Op.EOFCREATE]`.

Reply:
I also think it might not be as seamless to use this marker as is right now, it might need to require to move this test to another file to make use of different fixtures perhaps.

Reply:
Yup, I have seen it coming, which is also why I didn't want to overthink this for now. Would you like me to do this moving/splitting in this PR or leave it for after when that other marker merges and "stabilizes"?